src/main/resources/google_checks.xml

```diff
@@ -96,7 +96,7 @@
       <property name="tokens"
                value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT,
                     INSTANCE_INIT, ANNOTATION_DEF, ENUM_DEF, INTERFACE_DEF, RECORD_DEF,
-                    COMPACT_CTOR_DEF, LITERAL_SWITCH"/>
+                    COMPACT_CTOR_DEF, LITERAL_SWITCH, LITERAL_CASE"/>
```

Comment:
Show proof from google's style guide that this is the correct setting for cases.

Reply:
I added it in favor of  [rule 4.1.2](https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style) and I found nothing says that this rule should not be applied for cases

Reply:
If Google style guide is slowing down acceptance of this PR, I am good to make an issue to able it separately @rnveach

Reply:
I am good here. I question if us forcing `alone` violates https://google.github.io/styleguide/javaguide.html#s4.1.3-braces-empty-blocks if it is empty, but this would be more than just this 1 token impacted.